tests/numpy/manipulation_functions_test.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda.categorical import Categorical
+from arkouda.testing import assert_equal
+
+seed = pytest.seed
+
+
+class TestJoin:
```

Comment:
Im guessing this wasn't the intended class name. unfortunately it seems `TestManipulationFunctions` is already taken

Reply:
I believe that was a copy-and-paste error.  Thanks for catching it!  It should be fixed now.